samples/Kconfig

```diff
@@ -16,6 +16,15 @@ config NCS_SAMPLES_DEFAULTS
 	help
 	  Use the default configuration for NCS samples.
 
+config NCS_CUSTOM_RPMSG_CHILD_IMAGE
```

Comment:
This is used for both `RPMSG` and `802154_SER_HOST` its name should be more generic.

Reply:
What about making it generic for any child_image for the network core?

Reply:
so would NCS_CUSTOM_NETWORK_CHILD_IMAGE be good ? My intention was to override any auto child-image selection done by the NCS build system so it does make sense to remove the rpmsg bit

Reply:
IMO it would be best to have a config `AUTO_INCLUDE_NET_CORE_CHILD_IMAGE` or something like that, which was `default y`. Then the check in cmake looks a bit cleaner (if auto include AND something else), and to stop inclusion you would just unset it in your prj.conf.

Reply:
yes I tried that first, but I need this to compile a custom hci_rpmsg in the test repo. And since you can force a kconfig option ON but you can't force it OFF from a Kconfig file, then I moved to this solution.

Reply:
It should be possible to  force it OFF from a kconfig file by overriding the definition, just place the folliwing in one of your kconfig files:

```
config SAME_AS_THE_OTHER_ONE
    default n
```

And that should override the declaration and make it default n

Reply:
okay I just tried that but:
1. it seems pretty hack-ish to me an unclean
2. seems like it messed up dependencies or something, some other kconfig options were not selected because of the override

I can just tell people to set it manually to n in the prj.conf, I don't think we can have a clean n-select in kconfig.
Is the name NCS_AUTO_INCLUDE_NET_CORE_IMAGE ok with you ?

Reply:
Kconfig overrides are used in many places, and is not considered hack-ish. It should not affect dependencies other than what is stated in the new definition.

That name sounds good :)